www/pages/stop/stop.html

```diff
@@ -8,7 +8,7 @@
   </ion-nav-buttons>
   <ion-content overflow-scroll="true">
     <ion-refresher pulling-text="Pull to refresh..." on-refresh="refresh()"></ion-refresher>
-    <label class="item item-input item-select item-positive">
+    <label aria-label="This page auto refreshes content. To stop this, navigate to Settings and change the Auto refresh timing to off"   class="item item-input item-select item-positive">
```

Comment:
Might even want this at the top above the heading so that it's the very first thing an aria user hears.